ui-admin/src/study/participants/KitStatusCell.tsx

```diff
@@ -12,19 +12,20 @@ export default function KitStatusCell(
   { kitRequest: KitRequest, infoPlacement?: Placement }
 ) {
   const infoTexts: Record<string, string> = {
-    'kit without label': 'Kit request has been received',
-    'queue': 'Shipping label has been printed',
-    'sent': 'Kit has been sent to the participant',
-    'received': 'Kit has been returned by the participant',
-    'error': 'There was a problem fulfilling this kit request',
-    'deactivated': 'Kit is deactivated'
+    'CREATED': 'Kit has not yet shipped',
```

Comment:
I think the original text is more accurate.
A workflow of request received -> preparing to ship -> sent... makes logical sense and is consistent with general on-line ordering practices most people are familiar with.

Reply:
Ah, I see your point.
Yup, that works or something like "... awaiting processing"